debian/control

```diff
@@ -3,14 +3,33 @@ Maintainer: Team Kano <dev@kano.me>
 Section: admin
 Priority: optional
 Standards-Version: 3.9.2
-Build-Depends: debhelper (>= 9), build-essential, pkg-config, libgtk2.0-dev,
-               lxpanel (>= 0.7.0), libfm-dev, libkdesk-dev, gettext
+Build-Depends:
+    debhelper (>= 9),
+    build-essential,
+    pkg-config,
+    libgtk2.0-dev,
+    lxpanel (>= 0.7.0),
+    libfm-dev,
+    libkdesk-dev,
+    gettext
 
 Package: kano-updater
 Architecture: any
-Depends: ${misc:Depends}, kano-toolset (>= 3.7.0-1), python, libkdesk-dev,
- gir1.2-gtk-3.0 (>= 3.10.2), kano-settings (>= 1.3-1), python-apt, schedtool,
- python-pip, kano-i18n (>= 3.5-1), jq, kano-init, kano-profile (>= 3.9.0)
+Depends:
+    ${misc:Depends},
+    kano-toolset (>= 3.7.0-1),
+    python,
+    libkdesk-dev,
+    gir1.2-gtk-3.0 (>= 3.10.2),
+    kano-settings (>= 1.3-1),
+    python-apt,
+    schedtool,
+    python-pip,
+    kano-i18n (>= 3.5-1),
+    jq,
+    kano-init,
+    kano-profile (>= 3.9.0),
+    kano-peripherals
```

Comment:
Should you not list a version of `kano-peripherals` which has the required functions? Peripherals might need a version bump.